lms/templates/courseware/proctored-exam-status.underscore

```diff
@@ -6,10 +6,16 @@
                 .replace(/>/g, '&gt;')
         }
     %>
-    <div class='exam-text'>
+    <div class='exam-text js-exam-text' data-show-long="true">
```

Comment:
I have the same question about `exam-test` as I do about `toggle-show-more` below.

Reply:
I verified that this class applied styles on my dev machine. I haven't touched the corresponding styles, though, so I'm confident that if there's an inconsistency between our devstacks that it's not the result of my diff.

Reply:
I was referring to whether both `exam-text` and `js-exam-text` were both necessary as classes. I did leave multiple comments on the `toggle-show-more` line, so I apologize that that was unclear. I'll resolve this comment.